### PR TITLE
adding PIGroups to user management display

### PR DESCRIPTION
**Pull request by mckeags7**

### CODE REVIEW

**Comment by hakasapl** [comment]
Suggestion:
```
$cur_user_groups = $user->getGroups()
foreach ($cur_user groups as $cur_group) {
echo "<a href='mailto:" . $cur_group->getOwner()->getMail() . "'>" . $cur_group->getPIUID() . "</a>";
}
```

File: webroot/admin/user-mgmt.php [hunk 1]
```diff
@@ -42,9 +43,17 @@
...
         echo "<td>" . $user->getOrg() . "</td>";
-        echo "<td><a href='mailto:" . $user->getMail() . "'>" . $user->getMail() . "</a></td>";
+        echo "<td><a href ='mailto:" . $user->getMail() . "'>" . $user->getMail() . "</a></td>";
+        echo "<td>";
+        $cur_user_groups = $user->getGroups();
+        foreach($cur_user_groups as $cur_group) {
+            echo "<a href='mailto:" . $cur_group->getOwner()->getMail() . "'>" . $cur_group->getPIUID() . "</a>";
+            if ($cur_group !== array_key_last($cur_user_groups)) {
+                echo '<br>';
+        }
```
Comment (hakasapl) [inline webroot/admin/user-mgmt.php:53]:
```suggestion
            }
```
Really small change, just make sure these brackets line up with each other

File: webroot/admin/user-mgmt.php [hunk 2]
```diff
@@ -54,7 +63,9 @@
         echo "</td>";
         echo "</tr>";
     }
+
```
Comment (hakasapl) [inline webroot/admin/user-mgmt.php:66]:
```suggestion

```

File: webroot/admin/user-mgmt.php [hunk 3]
```diff
@@ -54,7 +63,9 @@
         echo "</td>";
         echo "</tr>";
     }
+    
     ?>
+
```
Comment (hakasapl) [inline webroot/admin/user-mgmt.php:68]:
```suggestion

```

**Review by hakasapl — CHANGES_REQUESTED** [review]
Some really small readability changes, otherwise the code looks good!

File: webroot/admin/user-mgmt.php [hunk 4]
```diff
@@ -42,9 +43,18 @@
...
+        echo "<td><a href ='mailto:" . $user->getMail() . "'>" . $user->getMail() . "</a></td>";
         echo "<td>";
+        $cur_user_groups = $user->getGroups();
+        foreach($cur_user_groups as $cur_group) {
+        echo "<a href='mailto:" . $cur_group->getOwner()->getMail() . "'>" . $cur_group->getPIUID() . "</a>";
+        if ($cur_group !== array_key_last($cur_user_groups)) {
+        echo '<br>';
+            }
 
+        }
```
Comment (hakasapl) [inline webroot/admin/user-mgmt.php:55]: Small indenting fix

```suggestion
        foreach($cur_user_groups as $cur_group) {
            echo "<a href='mailto:" . $cur_group->getOwner()->getMail() . "'>" . $cur_group->getPIUID() . "</a>";
            if ($cur_group !== array_key_last($cur_user_groups)) {
                echo '<br>';
            }
        }
```

File: webroot/admin/user-mgmt.php [hunk 5]
```diff
@@ -42,9 +43,18 @@
         echo "<td>" . $user->getFirstname() . " " . $user->getLastname() . "</td>";
         echo "<td>" . $user->getUID() . "</td>";
         echo "<td>" . $user->getOrg() . "</td>";
-        echo "<td><a href='mailto:" . $user->getMail() . "'>" . $user->getMail() . "</a></td>";
+        echo "<td><a href ='mailto:" . $user->getMail() . "'>" . $user->getMail() . "</a></td>";
```
Comment (hakasapl) [inline webroot/admin/user-mgmt.php:46]: Getting rid of extra space on the a tag

```suggestion
        echo "<td><a href='mailto:" . $user->getMail() . "'>" . $user->getMail() . "</a></td>";
```

File: webroot/admin/user-mgmt.php [hunk 6]
```diff
@@ -53,9 +63,9 @@
         </form>";
         echo "</td>";
         echo "</tr>";
+
```
Comment (hakasapl) [inline webroot/admin/user-mgmt.php:66]: Remove this line since it adds nothing

```suggestion

```

**Review by hakasapl — CHANGES_REQUESTED** [review]
This should be the last of the syntax fixes, once these are fixed we should be all set

**Review by mckeags7 — COMMENTED** [review]
approved changes

**Review by mckeags7 — COMMENTED** [review]
approved changes

**Review by hakasapl — APPROVED** [review]
Looks good to me!

**Pull request merged**